### DISCUSSION TITLE
EACCES Premision Denied Open '/home/wiki/data/cache/123.bin'

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3934] [Help / Questions]
Hello,
I'm getting this error here:
`EACCES: permission denied, open '/home/wikijs/data/cache/7f75d0584938f1bdf54b3a8bfd34e6cb7c61edd7.bin'`

I can't open the new created page. It is created but I'm unable to open it, I get this error.

Centos 8
SSL Lets Encrypt
Nginx

**Comment by snarum** [comment]
did you manage to fix this problem? I experience the same when running as systemd service. Tried to delete the cache file, but now I cant get wiki.js to run at all...

**Reply by ghost** [reply]
I don't remember but EACCES means there is a permission issue. Not sure if it was a group problem or permission (chmod) one.

**Reply by snarum** [reply]
I reinstalled everything and chown'ed all files and folders to the same user I have in my systemd file and now it works like a charm.

**Comment by HerbertNagy** [comment]
sudo chmod 777 -R /home/wikijs
